virtualcluster/config/rbac/role.yaml

```diff
@@ -0,0 +1,128 @@
+
```

Comment:
Apparently the `role.yaml` wasn't working cause the comments were nested under a function. Moving them allows controller-gen to regenerate these, I'm not sure if they are 100% but this work can be punted to #130